app/src/androidTest/java/org/mozilla/fenix/ui/DownloadTest.kt

```diff
@@ -40,28 +38,25 @@ class DownloadTest {
     @get:Rule
     val activityTestRule = HomeActivityIntentTestRule()
 
-    @Rule
-    @JvmField
-    val retryTestRule = RetryTestRule(3)
-
-    @get:Rule
-    var mGrantPermissions = GrantPermissionRule.grant(
-        android.Manifest.permission.WRITE_EXTERNAL_STORAGE,
-        android.Manifest.permission.READ_EXTERNAL_STORAGE
-    )
-
     @Before
     fun setUp() {
         // disabling the jump-back-in pop-up that interferes with the tests.
         featureSettingsHelper.setJumpBackCFREnabled(false)
         // disabling the PWA CFR on 3rd visit
         featureSettingsHelper.disablePwaCFR(true)
+        // clear all existing notifications
+        notificationShade {
+            mDevice.openNotification()
+            clearNotifications()
+        }
     }
 
     @After
     fun tearDown() {
-        deleteDownloadFromStorage(downloadFile)
         featureSettingsHelper.resetAllFeatureFlags()
+        notificationShade {
+            closeNotificationShade()
+        }
```

Comment:
This should already be done in the ActivityTestRule's afterActivityFinished()

Reply:
The function I'm using cancels all previously shown (clear), the other just closes the shade. Noted this is unideal. I'll rename my function.